### PR TITLE
Update example to a full functional application

### LINKED ISSUE #19: Update example
An issue might not be the best way, but it's just to let you know that I'm currently working on the example.

The idea is to have a full example to play with (create, update, delete).

### PR DESCRIPTION
**Pull request by stephanebachelier**
Close #19

### CODE REVIEW

File: examples/main.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
+(function(Backbone, $, _) {
+    'use strict';
+
+    // Set driver (optional, but we use localforage here so developers
```
Comment (tofumatt) [inline examples/main.js]: Should be "use localStorage" here, I think. Though Firefox devtools has an indexeddb storage inspector that's coming along awesome ;-)

Comment (tofumatt) [inline examples/main.js]: Should be "use localStorage" here, I think. Though Firefox devtools has an indexeddb storage inspector that's coming along awesome ;-)

**Comment by stephanebachelier** [comment]
@tofumatt if you want to play with the example it's here:

http://stephanebachelier.github.io/localForage-backbone/

**Comment by tofumatt** [comment]
Nice! I'll do a bit more review later this evening. That's awesome though
:-)

-tofumatt

**Comment by stephanebachelier** [comment]
this is localforage which is awesome

**Comment by tofumatt** [comment]
`examples/img/device.png` is cute but obviously an iPhone, any chance for a more generic phone outline graphic?

**Comment by tofumatt** [comment]
Looks like the icon set you're using isn't loading? I see black boxes in Safari and Unicode weird characters in Firefox (both Mac 10.10.2)

Safari:
screenshot 2015-03-10 17 24 19

Firefox (Developer Edition):
screenshot 2015-03-10 17 24 34

I love the different driver selection in the tabs though, that's very cool.

**Comment by tofumatt** [comment]
Overall great! Really cool, and I think I'll use this as a general localForage example too!

Can we fix the unicode bugs and possibly use a more generic phone graphic? If so I'd say it's good to merge!

**Comment by stephanebachelier** [comment]
@tofumatt I think the unicode bugs is due to CORS. I'll come back to you soon. For the phone I've used an iPhone because I add this image, but a Firefox OS phone image would be better :)

**Comment by stephanebachelier** [comment]
@tofumatt I've fixed the problem about the icon and change the phone image.

I've updated both this PR and the gh-pages which means that you can verify the changes on http://stephanebachelier.github.io/localForage-backbone/.
Tested with Safari, Chrome and FF dev edition.

I'll publish a mobile application based on this example next week on the stores.

**Comment by tofumatt** [comment]
Very cool! :+1:

**Pull request merged**